client/src/components/core/ms-dropdown/MsDropdown.vue

```diff
@@ -109,16 +109,6 @@ function getIcon(): string {
   }
 }
 
-.option {
-  --background-hover: var(--parsec-color-light-primary-50);
-  --color-hover: var(--ion-color-tertiary);
-
-  &.selected {
-    color: var(--parsec-color-light-primary-700);
-    font-weight: bold;
-  }
-}
-
```

Comment:
why remove this?

Reply:
I removed this code because it was unused.